core/App/ehsm_provider.cpp

```diff
@@ -939,6 +941,8 @@ ehsm_status_t GenerateQuote(ehsm_data_t *quote)
 }
 
 ehsm_status_t VerifyQuote(ehsm_data_t *quote,
+            string &mr_signer,
```

Comment:
mr_signer -> mrsigner_base
mr_enclave -> mrenclave_base

Reply:
I will change the name of the function parameter as you said.